vgm.sh

```diff
@@ -0,0 +1,35 @@
+package: VGM
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: "4.3"
+source: https://github.com/alisw/vgm.git
```

Comment:
Fixed ( Although what is the convention here? Many *.sh contain package names in capital letters ).